patches/minecraft/net/minecraft/world/level/levelgen/structure/pools/JigsawPlacement.java.patch

```diff
@@ -0,0 +1,73 @@
+--- a/net/minecraft/world/level/levelgen/structure/pools/JigsawPlacement.java
++++ b/net/minecraft/world/level/levelgen/structure/pools/JigsawPlacement.java
+@@ -44,6 +_,10 @@
+    static final Logger f_210265_ = LogUtils.getLogger();
+ 
+    public static Optional<PieceGenerator<JigsawConfiguration>> m_210284_(PieceGeneratorSupplier.Context<JigsawConfiguration> p_210285_, JigsawPlacement.PieceFactory p_210286_, BlockPos p_210287_, boolean p_210288_, boolean p_210289_) {
++      return addPieces(p_210285_, com.google.common.collect.Maps.newHashMap(), p_210286_, p_210287_, p_210288_, p_210289_);
++   }
++
++   public static Optional<PieceGenerator<JigsawConfiguration>> addPieces(PieceGeneratorSupplier.Context<JigsawConfiguration> p_210285_, java.util.Map<ResourceLocation, Object> placerSettings, JigsawPlacement.PieceFactory p_210286_, BlockPos p_210287_, boolean p_210288_, boolean p_210289_) {
```

Comment:
Overloads with additional context should have the new arguments added at the END of the method, not anywhere else.